src/scripts/main.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+  const notificationTitle = notification.appendChild(
+    document.createElement('h1'),
```

Comment:
The comma at the end of this line is unnecessary and could lead to confusion. Consider removing it to avoid potential syntax errors.